Sources/MarathonCore/Command.swift

```diff
@@ -90,6 +93,8 @@ extension Command {
             return ""
         case .help:
             return ""
+        case .export:
+            return "<script-path> [<export-path>] [--force]"
```

Comment:
Yeah usually the argument(s) need to directly follow the command name (see `Create.swift`, `Edit.swift`, for example), while the order of the flags doesn't matter.